autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/importing/takeout/zip/TakeoutBookmarkExtractor.kt

```diff
@@ -77,69 +80,66 @@ class TakeoutZipBookmarkExtractor @Inject constructor(
 
         val chromeHtmlFiles = mutableListOf<String>()
 
-        // First pass: collect Chrome HTML files and look for Bookmarks.html
```

Comment:
Need to update this so that it doesn't return early upon finding an exact match on the `Takeout/Chrome/Bookmarks.html` file since if there are more files in there, it is pixelled